### ISSUE TITLE
Migrate from within a seeder

### ISSUE THREAD

**Issue by ghost** [cakephp/phinx#720]
Seeds are database version specific.

Example:
DB-Version 1:
Table 'user' has columns 'id' and 'name'

Seed 1:
Add rows:
INSERT INTO 'user' ('id','name') VALUES (22, 'Fester Bester Tester')

DB-Migrate to version 2:
$table = Table('user')
$table->renameColumn('name','fullname');

From this point forward, Seed 1will fail, since 'user' no longer contains a column called 'name'.

I'm trying to extend the AbstractSeed class to include a method called migrateTo($target). This is really hard to do in a consistent way, since I can't get any information about the environment from within the Seed class because of the way its encapsulated.

**Comment by dereuromark** [comment]
What do you propose to be done here?
Always running seeds in parallel with its migration counterpart (timestamp wise)?

**Comment by othercorey** [comment]
Closing due to lack of activity or out of scope. If there is new interest, please open a PR.
